src/Service.Tests/GraphQLBuilder/MutationBuilderTests.cs

```diff
@@ -1079,7 +1095,7 @@ string[] expectedNames
             int totalExpectedMutations = 0;
             foreach ((_, DatabaseType dbType) in entityNameToDatabaseType)
             {
-                if (GraphQLUtils.DoesRelationalDBSupportMultipleCreate(dbType))
+                if (isMultipleCreateOperationEnabled && GraphQLUtils.DoesRelationalDBSupportMultipleCreate(dbType))
```

Comment:
nit: once `GraphQLUtils.DoesRelationalDBSupportMultipleCreate(dbType)` becomes implementation detail of `isMultipleCreateOperationEnabled` this can be modified.

Reply:
I've re-written this